test/snippet/alignment/pairwise/align_pairwise.cpp

```diff
@@ -20,4 +40,7 @@ int main()
     // Compute the alignment over a range of pairs.
     for (auto const & res : seqan3::align_pairwise(vec, seqan3::align_cfg::edit))
         seqan3::debug_stream << "The score: " << res.score() << "\n";
+    //![example3]
+//![end]
```

Comment:
I'm confused that this works without a `return 0;` statement at the end of the main function... However, if Travis is green, this should be fine.

Reply:
As far as I know, main() is the only function that doesn't need a return statement, even though it's not void.